marshal.go

```diff
@@ -2352,7 +2370,7 @@ func (inetType) Type() Type {
 
 // Zero returns the zero value for the inet CQL type.
 func (inetType) Zero() interface{} {
-	return net.IP(nil)
+	return net.IP{}
```

Comment:
What's this change for? net.IP is a slice and the zero value for a slice is nil.

Reply:
I went back and forth on this and I forgot to revert back to `nil` good catch